components/org.wso2.carbon.identity.data.publisher.authentication.analytics.login/src/test/java/org/wso2/carbon/identity/data/publisher/authentication/analytics/login/AnalyticsLoginDataPublisherUtilsTest.java

```diff
@@ -62,6 +64,7 @@ public Object[][] getEvent() {
         Map<String, Object> param = new HashMap<>();
         Object userObj = new AuthenticatedUser();
         ((AuthenticatedUser) userObj).setTenantDomain(TENANT_DOMAIN);
+        ((AuthenticatedUser) userObj).setUserId(USER_ID);
```

Comment:
why is this needed in the test?

Reply:
Otherwise this test is failing due to a NPE.

Reply:
the underlying cause was due to the pax-logger. fixed it.